build.gradle

```diff
@@ -415,6 +415,10 @@ bootJar {
   }
 }
 
+jar {
+  enabled = false
```

Comment:
this was done as spring 2.5.x creates a plain-jar file was well as the actual jar file. we can do this or hard code the repo name in the docker file here: https://github.com/hmcts/hmc-hmi-outbound-adapter/blob/408e8877da2d834325bef50324816e653d752c61/Dockerfile#L4